### PR TITLE
Add version guards & restore the old behaviour for versions prior to Houdini 16.0

### PR DESCRIPTION
**Pull request by jmlait**

### CODE REVIEW

File: openvdb_houdini/houdini/GEO_VDBTranslator.cc [hunk 1]
```diff
@@ -231,14 +236,11 @@ GEO_VDBTranslator::fileStat(const char *filename, GA_Stat &stat, uint level)
 
     return true;
 }
+#endif
```
Comment (danrbailey) [inline openvdb_houdini/houdini/GEO_VDBTranslator.cc]: This endif is un-paired, I've added a pull request into sideeffects/send_upstream that should fix this:

https://github.com/sideeffects/openvdb_dev/pull/1

**Comment by danrbailey** [comment]
@OpenVDB-DevTeam @nyue - this pull request is still not working for me, I've submitted a patch to sideeffects/send_upstream (https://github.com/sideeffects/openvdb_dev/pull/1)

File: openvdb_houdini/houdini/GEO_VDBTranslator.cc (hunk 1 above)
Comment (jmlait) [inline openvdb_houdini/houdini/GEO_VDBTranslator.cc]: Thank you.  Merged to sideeffects/send_upstream

Comment (danrbailey) [inline openvdb_houdini/houdini/GEO_VDBTranslator.cc]: Thanks Jeff!

**Comment by danrbailey** [comment]
Edit: all working now, this is good to be merged.

**Pull request merged**